Ska/engarchive/fetch.py

```diff
@@ -69,6 +70,31 @@
 # Cached version (by content type) of first and last available times in archive
 CONTENT_TIME_RANGES = {}
 
+# Data source. Default to CXC. Set to MAUDE by fetch.from_maude()
+_backend = 'CXC'
+
+def from_cxc():
+    maude.handler.disconnect()
+    global _backend
+    _backend = 'CXC'
+
+
+def from_maude(channel='FLIGHT'):
+    """
+    Attempt to connect to MAUDE. Set the backend flag to 'MAUDE' if successful.
+
+    :param channel: MAUDE telemetry channel (FLIGHT, FLTCOMP, ASVT, TEST)
+    """
+    try:
+        maude.handler.connect()
```

Comment:
Could this connect go into the `handler.get()` method so the connect is not attempted until actually trying to fetch telemetry?